contrib/scripts.py

```diff
@@ -9,15 +9,16 @@
 from typing import TYPE_CHECKING
 
 import requests
-from beancount.query import query_env
-from beancount.query import query_parser
+from beanquery import query_compile
```

Comment:
`beanquery.query_compile` is an implementation detail. There is no guarantee that anything in there is not going to change without warning.

Reply:
this file `contrib/scripts` generates the JSON file for the frontend query autocompletion, which only needs to run with a fixed version of beanquery (constraints.txt) at irregular points in time, so I'm fine with using internal implementation details here